### PR TITLE
enable requestUUID

### PR DESCRIPTION
**Pull request by jingweicb**
Fixes # .
use two methods to add requestUUID
1, add it into ctx
2, add it to logger

Screenshot 2022-12-08 at 12 17 56 AM

Screenshot 2022-12-08 at 12 19 26 AM

### CODE REVIEW

File: pkg/logger/logger.go [hunk 1]
```diff
@@ -94,9 +99,10 @@ func NewLogger(
 	logReconciliation bool,
 	checkType CheckType,
 	network *types.NetworkIdentifier,
+	logRequestUUID string,
 	fields ...zap.Field,
 ) (*Logger, error) {
-	zapLogger, err := buildZapLogger(checkType, network, fields...)
+	zapLogger, err := buildZapLogger(checkType, network, logRequestUUID, fields...)
```
Comment (jingweicb) [inline pkg/logger/logger.go:105]: @shrimalmadhur,  this part will make requestUUID to be a field in datadog?

Comment (shrimalmadhur) [reply pkg/logger/logger.go:105]: At least this should make the field as structured - we need to make sure the logs are emitted to datdog agent and not picked up via stdout

Comment (shrimalmadhur) [reply pkg/logger/logger.go:105]: @jingweicb we need to change this to Production logger - https://github.com/coinbase/rosetta-cli/blob/master/pkg/logger/logger.go#L118

**Review by xiaying-peng — APPROVED** [review]

File: pkg/logger/logger.go [hunk 2]
```diff
@@ -279,10 +295,11 @@ func (l *Logger) RemoveBlockStream(
 	defer closeFile(f)
 
 	blockString := fmt.Sprintf(
-		"%s Block %d:%s\n",
+		"%s Block %d:%s , RequestUUID: %s\n",
```
Comment (xiaying-peng) [inline pkg/logger/logger.go:298]: no space!

**Review by xiaying-peng — APPROVED** [review]

**Pull request merged**